site/data/records.json

```diff
@@ -750,5 +750,27 @@
         "license": "MIT",
         "upload_date": "03/11/2023",
         "rank": 0.99
+    },
+    {
+        "name": "BotInfomoney",
+        "description_short": "The automation collects information on the Infomoney website, searches the index and shares value on Google finance, and generates a demonstrative spreadsheet.",
+        "description": "The automation collects information on the most significant rises in the stock market through the Infomoney website, searches the index and value of shares on Google Finance, and generates a demonstrative spreadsheet.",
+        "tags": ["python", "botcity", "rpa"],
+        "programming_language": "python",
+        "technology": ["botcity"],
+        "OS": ["Windows", "Linux"],
+        "thumbnail_url": "https://i.imgur.com/vbzzebw.png",
```

Comment:
Maybe we could generate a thumbnail without BotCity logo and more related to the bot subject. That would likely attract more people's attention to the bot.

Reply:
Nice! I changed to another image:
![bot image](https://github.com/botcity-dev/bot-repository/assets/19210643/146d40c5-7662-4657-b1fd-a3a263c22b89)